TombEngine/Specific/level.h

```diff
@@ -137,15 +137,16 @@ struct LEVEL
 
 extern std::vector<int> MoveablesIds;
 extern std::vector<int> StaticObjectsIds;
-extern bool IsLevelLoading;
 extern LEVEL g_Level;
 
+inline std::future<bool> level_load_ft;
```

Comment:
Can we use different naming convention here? I recommend using PascalCase.

Reply:
Fixed